wallet/src/bin/faucet_server.rs

```diff
@@ -0,0 +1,163 @@
+// Copyright (c) 2022 Espresso Systems (espressosys.com)
+// This file is part of the Configurable Asset Privacy for Ethereum (CAPE) library.
+
+// This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+// This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+// You should have received a copy of the GNU General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+//! # The CAPE Faucet
+//!
+
+extern crate cape_wallet;
+
+use async_std::{
+    sync::{Arc, Mutex},
+    task::{spawn, JoinHandle},
+};
+use cape_wallet::{
+    routes::{init_wallet, Wallet},
+    wallet::CapeWalletError,
+    web::default_storage_path,
+};
+use jf_cap::{
+    keys::{UserKeyPair, UserPubKey},
+    structs::AssetCode,
+};
+use rand_chacha::{rand_core::SeedableRng, ChaChaRng};
+use seahorse::loader::Loader;
+use serde::{Deserialize, Serialize};
+use snafu::Snafu;
+use std::path::PathBuf;
+use structopt::StructOpt;
+use tide::StatusCode;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "CAPE Faucet Server",
+    about = "Grants a native asset seed to a provided UserPubKey"
+)]
+pub struct FaucetOptions {
+    /// mnemonic for the faucet wallet
+    #[structopt(long = "mnemonic", env = "MNEMONIC", default_value = "")]
```

Comment:
I will suggest some more explicit env vars: `CAPE_FAUCET_WALLET_MNEMONIC`?

For the others, could we add the `CAPE_` prefix?

Reply:
Sure, that's reasonable.